src/diff/amfGraphDifferencer.ts

```diff
@@ -171,6 +171,17 @@ function parseGraphDelta(
   return graphChanges;
 }
 
+/**
+ * Returns true if type of amf graph node is defined
+ * @param nodeType - Types of amf graph node
+ */
+function isNodeTypeDefined(
+  nodeType: string[] | undefined
+): nodeType is string[] {
```

Comment:
change return type to boolean

Reply:
https://www.typescriptlang.org/docs/handbook/advanced-types.html